scripts/delete-old-app-engine-versions.sh

```diff
@@ -45,10 +45,10 @@ abort() {
     exit 1
 }
 
-# ensure that date from GNU coreutils is installed
-check_gnu_date_installed() {
-    if ! date --version 1>/dev/null 2>&1; then
-        abort "date from GNU coreutils is required"
+# ensure that jq is installed
+check_jq_installed() {
+    if ! jq --version 1>/dev/null 2>&1; then
+        echo "jq v1.6 or above is required"
```

Comment:
Maybe add `brew install jq` to the error message?

Reply:
Can we guarantee that everyone who runs this is using MacOS? I'm not sure.

Reply:
I personally like error messages with the install step as well. Given that it is a shell script, I think adding a `brew install jq` message is ok. You can run install and run brew on Linux as well.

Reply:
I added this to the error message.